### PR TITLE
Use summary list for key facts, heading semantics etc

### PR DESCRIPTION
**Pull request by colinrotherham**
Just a few things I spotted whilst reviewing https://github.com/alphagov/govuk-design-system/pull/3177 to help screen readers out

@owenatgov @stevenjmesser Mind taking a look at this PR for me please?

* Move key facts into `<dl>` using the [**Summary list** component](https://design-system.service.gov.uk/components/summary-list/)
* Mark up "**In the morning**" and "**After lunch**" as headings

I've also run `prettier --write` on the project as the optional check was getting a bit shouty

Summary list on tablet

<br>
<br>

Summary list on mobile

### CODE REVIEW

File: src/community/design-system-day/index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Most sessions are talks or show and tells where speakers may take questions at t
 
 #### In the morning
 
-- We’re feeling ‘appy
+- We’re feeling ’appy
```
Comment (colinrotherham) [inline src/community/design-system-day/index.md:100]: Text editors love flipping apostrophes the other way around 🤦‍♂️

Comment (stevenjmesser) [reply src/community/design-system-day/index.md:100]: Thank you!

**Review by stevenjmesser — APPROVED** [review]
Ah, the beauty of open source.

Thanks so much for suggesting these changes, it looks so much better. I did try a non-bulleted list for the information at the top but it erroneously rendered as bulleted. A summary list does the trick wonderfully.

**Comment by stevenjmesser** [comment]
As a possible further improvement, does the header image resolution need to increase to span the whole content width?

**Comment by colinrotherham** [comment]
@stevenjmesser Every little helps 🙌 

> As a possible further improvement, does the header image resolution need to increase to span the whole content width?

Happy to leave it, we keep the [722px max-width](https://github.com/alphagov/govuk-design-system/pull/970) elsewhere too:
https://design-system.service.gov.uk/community/#collaborate-and-co-design

**Pull request merged**